lib/block/header.ex

```diff
@@ -3,56 +3,76 @@ defmodule Block.Header do
   This structure codifies the header of a block in the blockchain.
   """
 
-  @empty_trie MerklePatriciaTree.Trie.empty_trie_root_hash
-  @empty_keccak [] |> ExRLP.encode |> :keccakf1600.sha3_256
-
-  defstruct [
-    parent_hash: nil,                # Hp P(BH)Hr
-    ommers_hash: @empty_keccak,      # Ho KEC(RLP(L∗H(BU)))
-    beneficiary: nil,                # Hc
-    state_root: @empty_trie,         # Hr TRIE(LS(Π(σ, B)))
-    transactions_root: @empty_trie,  # Ht TRIE({∀i < kBTk, i ∈ P : p(i, LT (BT[i]))})
-    receipts_root: @empty_trie,      # He TRIE({∀i < kBRk, i ∈ P : p(i, LR(BR[i]))})
-    logs_bloom: <<0::2048>>,         # Hb bloom
-    difficulty: nil,                 # Hd
-    number: nil,                     # Hi
-    gas_limit: 0,                    # Hl
-    gas_used: 0,                     # Hg
-    timestamp: nil,                  # Hs
-    extra_data: <<>>,                # Hx
-    mix_hash: nil,                   # Hm
-    nonce: nil,                      # Hn
-  ]
+  @empty_trie MerklePatriciaTree.Trie.empty_trie_root_hash()
+  @empty_keccak [] |> ExRLP.encode() |> :keccakf1600.sha3_256()
+
+            # Hp P(BH)Hr
```

Comment:
Is it possible to control the settings on the formatter?

Reply:
@hayesgm I did not find this setting in [official docs](https://hexdocs.pm/mix/master/Mix.Tasks.Format.html)